web/themes/custom/drevops/drevops.libraries.yml

```
@@ -1,10 +1,17 @@
   js:
-    dist/civictheme.js: {}
+    dist/scripts.drupal.base.js: {}
   dependencies:
     - core/drupal
     - core/once
     - core/drupalSettings
+
+css-variables:
+  css:
+    theme:
+      dist/styles.variables.css: { preprocess: false, weight: 10 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)